plug-ins/groups/group_combat.cpp

```diff
@@ -198,39 +223,60 @@ SPELL_DECL(DesertFist);
 VOID_SPELL(DesertFist)::run( Character *ch, Character *victim, int sn, int level ) 
 { 
         
-        int dam;
+    int dam;
 
-        if ( (ch->in_room->sector_type != SECT_HILLS)
-                && (ch->in_room->sector_type != SECT_MOUNTAIN)
-                && (ch->in_room->sector_type != SECT_DESERT) )
-        {
-                ch->println("Здесь недостаточно песка, чтобы сформировать кулак.");
-                ch->wait = 0;
-                return;
-        }
-        
-        act("Вихрь песка поднимается с земли, образуя огромный кулак, и ударяет $c4.", victim, 0, 0, TO_ROOM);
-        act("Вихрь песка поднимается с земли, образуя огромный кулак, и ударяет тебя.", victim, 0, 0, TO_CHAR);
-        dam = dice( level , 14 );
+    if (   (ch->in_room->sector_type != SECT_HILLS)
+        && (ch->in_room->sector_type != SECT_MOUNTAIN)
+        && (ch->in_room->sector_type != SECT_DESERT) )
+    {
+        ch->println("Здесь недостаточно песка, чтобы сформировать кулак.");
+        ch->wait = 0;
+        return;
+    }
 
-        damage_nocatch(ch,victim,dam,sn,DAM_OTHER,true, DAMF_SPELL);
-        sand_effect(victim,level,dam,TARGET_CHAR, DAMF_SPELL);
+    act_p("{yВихрь песка,{x созданный $c5, образует огромный кулак и ударяет $c4.",
+        ch,0,victim,TO_NOTVICT,POS_RESTING);
+    act_p("{yВихрь песка,{x созданный тобой, образует огромный кулак и ударяет $c4.",
+        ch,0,victim,TO_CHAR,POS_RESTING);
+    act_p("{yВихрь песка,{x созданный $c5, образует огромный кулак и ударяет тебя!",
+        ch,0,victim,TO_VICT,POS_RESTING);
+
+    // Tier 1 damage: only spells with severe limitations
+         if (level <= 20)   dam = dice( level, 10 );    
+    else if (level <= 40)   dam = dice( level, 13 );
+    else if (level <= 70)   dam = dice( level, 16 );
+    else                    dam = dice( level, 20 );
+
+    damage_nocatch(ch,victim,dam,sn,DAM_BASH,true, DAMF_SPELL);
+    sand_effect(victim,level,dam,TARGET_CHAR, DAMF_SPELL);
 }
 
 SPELL_DECL(EtheralFist);
 VOID_SPELL(EtheralFist)::run( Character *ch, Character *victim, int sn, int level ) 
 { 
     
     int dam;
+
+    act_p("Призванный $C5 кулак из иномирового эфира сокрушает $C4, повергая $S в ошеломление!",
```

Comment:
@fiorine may be it's better to only show "daze" messages when wait state is applied? Or have 2 sets of messages, one for damage+daze and another one for plain damage.